source/win-main/store/actions/filtree-update.ts

```diff
@@ -0,0 +1,109 @@
+/**
```

Comment:
Small typo in file name: Should be "filetree-update.ts", correct?

Reply:
Yup!